_includes/search.html

```diff
@@ -0,0 +1,20 @@
+<!-- Html Elements for Search -->
```

Comment:
Thank you @gabrielpadilh4 for the PR! 
Please feel free to go ahead and close the other PR to avoid confusion. 
Here are a few things I wanted to suggest: 
Is it possible to have the original list of blogs appear when the user first clicks on https://gabrielpadilh4.github.io/wildfly-elytron/blog/? It is a blank page for me with the tags on the right side and the search bar at the top. 
![image](https://github.com/wildfly-security/wildfly-elytron/assets/83880425/8ff47a87-de09-4d6a-a8ca-fa3123745cf0)

Is it possible to have the blogs that contain the search term listed as the results when a user presses enter? For example, let's say I type in "TLS", then all the blog posts that contain TLS in the title or the content would be displayed?  Currently nothing happens when I press enter. But if we type in TLS, then the following blog posts are suggested
![image](https://github.com/wildfly-security/wildfly-elytron/assets/83880425/020f677e-a627-4b89-b658-9057d9414beb)
Perhaps we can display those blog posts in a list once the user presses enter?

Reply:
Hello @PrarthonaPaul yeah, i'm trying to figure out why it is displaying a blank page as locally it shows the posts. I will work on that. 
The way that i have implemented, it generates a json with the terms that we can use for search like the title, tags, synopsis, etc. I don't know if this is possible, however, i will take a look and will keep you posted.

Thank you for the suggestions, i will work on that.

Reply:
Hello @PrarthonaPaul i did some update, could you please review and let me know your thoughts ?

Reply:
Thank you for making the updates. 
It looks good to me!